### ISSUE TITLE
How to use this ?

### ISSUE THREAD

**Issue by Uglysarthak** [Dropout1337/Discord-Mass-DM#10]
Your readme doesnt contain anything regarding the usage of the bot

**Comment by Pyreaus** [comment]
> 
> 
> Your readme doesnt contain anything regarding the usage of the bot

Exactly, I suppose there are some esoteric in-chat commands you give after inviting the bot to your server, but my guess is as good as any.

**Comment by dauntIess** [comment]
Just use the selfbot version, follow this video in order to find your discord token: https://www.youtube.com/watch?v=YEgFvgg7ZPI

When you have your token, open the source for selfbot.py and paste your token in line five where necessary. Save the file, now open selfbot.py file. Then type **gay!dmall <YOURMESSAGE>** in the required server. This might take a while for it to dm everyone because of discord's rate limits

**Comment by AdamT20054** [comment]
How do you choose what the bot messages them?
